test/testutils.jl

```diff
@@ -17,7 +23,7 @@ function test_pdmat(C, Cmat::Matrix;
                     t_cholesky::Bool=true,      # whether to test cholesky method
                     t_scale::Bool=true,         # whether to test scaling
                     t_add::Bool=true,           # whether to test pdadd
-		    t_det::Bool=true,           # whether to test det method
+		            t_det::Bool=true,           # whether to test det method
```

Comment:
This seems to be an accident.

Reply:
I noticed the change in my git diff but in my editor the spacing actually looked correct with the change and wrong on the master branch. Weird, I'll fix it.

Reply:
Seems the file contained a few tabs. Should be fixed now.